products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+## Game of Thrones deaths
+
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
+### Introduction
+
+**WARNING** - If you've been living under a rock and still have not watched Game of Thrones,
+this tutorial could contain ***spoilers***. 
+
+We can all agree that 2020. was one of the worst years in our lives because of the worldwide
+pandemic, 2019. still comes close with that disappointing, poorly written ending to one of the
+best shows of all time, Game of Thrones. Seven-year build-up to that awful last season crushed
+the souls of millions of fans around the world. Nevertheless, the Game of Thrones world is full
+of interesting characters, locations, scenarios, and most importantly deaths. If you want to know

```

Comment:
`[...] and most importantly deaths.` It seems like that was the most important thing in GoT, but it isn't. Maybe rewrite this in such a sense where you reference sudden deaths like Ned Stark and how it changed that feeling that main characters will be alive till the end so on so on...